static/app/utils/performance/suspectSpans/suspectSpansQuery.tsx

```diff
@@ -21,10 +23,19 @@ type Props = RequestProps & {
   children: (props: ChildrenProps) => ReactNode;
 };
 
+function getSuspectSpanPayload(props: RequestProps) {
+  const payload = {spanOp: props.spanOps};
+  const additionalPayload = omit(props.eventView.getEventsAPIPayload(props.location), [
+    'field',
```

Comment:
Why are you omitting field here?

Reply:
`field` is not used in this endpoint at all as they are predetermined in the endpoint.